dolphinscheduler-task-plugin/dolphinscheduler-task-tis/src/test/resources/org/apache/dolphinscheduler/plugin/task/tis/TISTaskTest.json

```diff
@@ -0,0 +1,60 @@
+[
+  {
+    "description": "trigger TIS task execute",
+    "request": {
+      "uri": "/tjs/coredefine/coredefine.ajax",
+      "method": "post",
+      "headers": {
```

Comment:
Seems to have lost the license head

Reply:
how to add license in json format file, as i have review some existing json format file in project  DS have not found license annotation ever

Reply:
sorry,I made a mistake, thank you for your reminder.